JsonSchema/nuspec/JsonSchema.Net.it.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.12">
+    <id>JsonSchema.Net.it</id>
+    <version>2.0.0</version>
```

Comment:
Version should be 1.0.0 since this will be a new package.

Reply:
You're right! I have updated the PR changing the version from `2.00` to `1.0.0`.